src/RE/H/HUDNotifications.cpp

```diff
@@ -0,0 +1,16 @@
+#include "RE/H/HUDNotifications.h"
+
+namespace RE
+{
+	HUDNotifications::Notification::Notification(const Notification& a_rhs)
```

Comment:
```suggestion
```
It would be preferable not to create additional .cpp files and have this in the header instead.